src/app/routes/index.js

```diff
@@ -0,0 +1,26 @@
+import { Link, Route, Switch } from 'react-router-dom';
+import React from 'react';
+import ChannelsList from '../features/Chanels/ChannelsList';
+import Channel from '../features/Chanels/Channel';
+import 'antd/dist/antd.css';
+
+const Routes = () => {
+  return (
+    <div>
+      <nav className="navbar navbar-dark bg-primary d-block">
```

Comment:
Consider creating Header (or Navigation) component and moving it to app.js. The idea is that nav bar may be extended with other control buttons, logo, etc.